src/ntfs/read.rs

```diff
@@ -314,8 +314,10 @@ pub fn manage_stop_times(collections: &mut Collections, path: &path::Path) -> Re
             .index_mut(vj_idx)
             .stop_times
             .push(crate::objects::StopTime {
+                id: None,
```

Comment:
I guess you're talking about the serialization where the name of the fields are [`stop_time_id` and `stop_headsign`](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md#stop_timestxt-requis)?  Yes, I think this will be addressed in a later PR because I'm not sure I can just use some nice `serde` macro or if I will need more work.